examples/VREP/dynamics_adaptation_osc.py

```diff
@@ -69,14 +71,12 @@
             target=target,
             )
 
-        scaled = robot_config.scaledown('q', feedback['q'])
-        u_adapt = adapt.generate(
-            input_signal=np.array([scaled[1], scaled[2]]),
+        u_adapt = np.zeros(robot_config.N_JOINTS)
+        u_adapt[1:3] = adapt.generate(
+            input_signal=np.array(feedback['q'][1], feedback['q'][2]),
```

Comment:
Doesn't this throw an error? `np.array(1, 2)` gives `TypeError: data type not understood`. Missing the `[]` brackets?

Reply:
damn, so this runs for me. i guess the second feedback value is in the proper range for another parameter? It would have taken a long time to find this, thanks for the catch!

Reply:
corrected in #47 and reported in numpy/numpy#13569